packages/types/src/documents/app/app.ts

```diff
@@ -73,3 +75,30 @@ export interface AppFeatures {
 export interface AutomationSettings {
   chainAutomations?: boolean
 }
+
+export interface CreateAppRequest {
```

Comment:
Generally we separate the `document` definitions and the API request definitions, these should probably be moved to the `types/src/api/web` layer.